src/web/mjs/engine/Settings.mjs

```diff
@@ -65,6 +67,18 @@ export default class Settings {
             input: types.directory,
             value: path.join( docs, 'Mangas' )
         };
+        let electron = require( 'electron' );
```

Comment:
You may place this at the top and then change the `app` assignment to use this variable instead to avoid double `require` statements
```javascript
let electron = require( 'electron' );
let app = electron.remote.app;
```

Reply:
ahh yes, app was already declared there, electron was probably automatically declared by the IDE
since you recommended that for the `getPath`